File: Core/Entities/MedicalRecord.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    public class MedicalRecord : BaseEntity
    {
        public string AnamnesisDiagnosisTherapy { get; set; }

        [DataType(DataType.Date)]
        public DateTime Created { get; set; }

        public int Patient1Id { get; set; }
        [ForeignKey("Patient1Id")]
        public Patient1 Patient { get; set; }

        public int Office1Id { get; set; }     
        [ForeignKey("Office1Id")]
        public Office1 Office { get; set; }     
    }
}

File: angular-project/src/app/shared/models/medicalrecord.ts
export interface IMedicalrecord {
    id: number;
    anamnesisDiagnosisTherapy: string;
    created: Date;
    doctor: string;
    patient: string;
    office: string;
    appointment1Id: number;

}

export class INewMedicalrecordToCreate {
    id: number;
    anamnesisDiagnosisTherapy: string;
    created: Date;
    patient1Id: number;
    office1Id: number;
}

export interface IMedicalrecord1 {
    id: number;
    anamnesisDiagnosisTherapy: string;
    created: Date;
    patient: string;
    doctor: string;
    office: string;
    appointment1Id: number;
}

export class INewMedicalrecordToCreate1 {
    anamnesisDiagnosisTherapy: string;
    created: Date;
    appointment1Id: number;
}
